pages/about.tsx

```diff
@@ -258,6 +258,7 @@ const AboutPage = () => {
               to be paid upon execution, measured in gas. This cost is the same
               for all executions, though this is subject to change in new
               hardforks. See our
+              {' '}
```

Comment:
I think it has to go on the same line.
```suggestion
              hardforks. See our{' '}
```